// signal-numbers/lib/index.js
"use strict";
//  Copyright 2018 Bryant Luk
//
//  Licensed under the Apache License, Version 2.0 (the "License");
//  you may not use this file except in compliance with the License.
//  You may obtain a copy of the License at
//
//  http://www.apache.org/licenses/LICENSE-2.0
//
//  Unless required by applicable law or agreed to in writing, software
//  distributed under the License is distributed on an "AS IS" BASIS,
//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
//  See the License for the specific language governing permissions and
//  limitations under the License.
Object.defineProperty(exports, "__esModule", { value: true });
const mapping = {
    SIGABRT: 6,
    SIGALRM: 14,
    SIGBUS: 10,
    SIGCHLD: 20,
    SIGCONT: 19,
    SIGFPE: 8,
    SIGHUP: 1,
    SIGILL: 4,
    SIGINT: 2,
    SIGKILL: 9,
    SIGPIPE: 13,
    SIGQUIT: 3,
    SIGSEGV: 11,
    SIGSTOP: 17,
    SIGTERM: 15,
    SIGTRAP: 5,
    SIGTSTP: 18,
    SIGTTIN: 21,
    SIGTTOU: 22,
    SIGUSR1: 30,
    SIGUSR2: 31,
};
function convert(signal) {
    return mapping[signal];
}
exports.convert = convert;
